packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -20,128 +20,57 @@ const INTERVAL_SECONDS = 1
 const MAX_ATTEMPTS = 20
 
 /**
- * The fundWallet() method is used to send an amount of XRP (usually 1000) to a new (randomly generated)
- * or existing XRP Ledger wallet.
+ * Generate a new wallet to fund if no existing wallet is provided or its address is invalid.
  *
- * @example
- *
- * Example 1: Fund a randomly generated wallet
- * const { Client, Wallet } = require('xrpl')
- *
- * const client = new Client('wss://s.altnet.rippletest.net:51233')
```

Comment:
I am not a fan of getting rid of these examples.  Are these being moved somewhere else?

Reply:
These examples live in the client folder the function `fundWallet` was moved on the class instead of in a separate file to allow the docs to follow the class method. The examples were kept.